src/dex/uniswap-v2/constants.ts

```diff
@@ -15,6 +16,7 @@ export const AllUniswapForks = [
   ...Object.keys(BiSwapConfig),
   ...Object.keys(DfynConfig),
   ...Object.keys(ExcaliburConfig),
+  ...Object.keys(DystopiaConfig),
```

Comment:
I think Dystopia should not be included in `AllUniswapForks` nor `UniswapForksWithNetwork` because it is not a standard fork, and it will break things if it gets mixed with another fork as `UniswapForkOptimized`.  Also, it doesn't support buy.  Comment it out on both and include a note in the comment above which currently covers `BakerySwap`.

Reply:
Fixed